xpdacq/beamtime.py

```diff
@@ -168,6 +168,65 @@ def Tramp(dets, exposure, Tstart, Tstop, Tstep, *, md=None):
     yield from plan
 
 
+def T_list(dets, exposure, T_list):
```

Comment:
To keep things consistent, should we call it `Tlist`?  Since we use `Tramp` and not `T_ramp`.